### DISCUSSION TITLE
Feature Request: NormaliseTag() feature to standardize tag format

### DISCUSSION THREAD

**Discussion by ziming** [spatie/laravel-tags#438] [Ideas]
It would be nice if there is a function we can define called normaliseTag() or something

Then inside there we can apply slug or lowercase or uppercase.etc to ensure all slugs are standardized before saving

**Comment by freekmurze** [comment]
You can achieve this by [using your own model](https://github.com/spatie/laravel-tags/blob/main/config/tags.php#L14) and add any function (or modify any function) your heart desires.
